src/Symfony/Component/HttpClient/Retry/GenericRetryStrategy.php

```diff
@@ -36,42 +36,35 @@ class GenericRetryStrategy implements RetryStrategyInterface
         510 => self::IDEMPOTENT_METHODS,
     ];
 
-    private array $statusCodes;
-    private int $delayMs;
-    private float $multiplier;
-    private int $maxDelayMs;
-    private float $jitter;
-
     /**
      * @param array $statusCodes List of HTTP status codes that trigger a retry
      * @param int   $delayMs     Amount of time to delay (or the initial value when multiplier is used)
      * @param float $multiplier  Multiplier to apply to the delay each time a retry occurs
      * @param int   $maxDelayMs  Maximum delay to allow (0 means no maximum)
      * @param float $jitter      Probability of randomness int delay (0 = none, 1 = 100% random)
      */
-    public function __construct(array $statusCodes = self::DEFAULT_RETRY_STATUS_CODES, int $delayMs = 1000, float $multiplier = 2.0, int $maxDelayMs = 0, float $jitter = 0.1)
-    {
-        $this->statusCodes = $statusCodes;
-
+    public function __construct(
+        private array $statusCodes = self::DEFAULT_RETRY_STATUS_CODES,
+        private int $delayMs = 1000,
```

Comment:
I would revert this change as well. In other places where we check the passed value to (for example) throw exceptions we don't use CPP.

Reply:
Done